invenio_records_resources/services/records/service.py

```diff
@@ -218,15 +218,17 @@ def reindex(self, identity, params=None, es_preference=None, **kwargs):
         self.indexer.bulk_index(iterable_ids)
         return True
 
-    def create(self, identity, data):
+    @unit_of_work()
+    def create(self, identity, data, uow=None):
         """Create a record.
 
         :param identity: Identity of user creating the record.
         :param data: Input data according to the data schema.
         """
-        return self._create(self.record_cls, identity, data)
+        return self._create(self.record_cls, identity, data, uow=uow)
 
-    def _create(self, record_cls, identity, data, raise_errors=True):
+    @unit_of_work()
```

Comment:
This is here, so that we're sure we always have a unit of work.....it will do nothing when called from ``create()`` above.